azkaban-web-server/src/test/java/azkaban/fixture/MockLoginAzkabanServlet.java

```diff
@@ -107,7 +108,7 @@ public static MockLoginAzkabanServlet getServletWithSession(final String session
 
     // Create a concrete SessionCache so a session will get persisted
     // and can get looked up
-    final SessionCache cache = new SessionCache(props);
+    final SessionCache cache = new SessionCache(props, null);
```

Comment:
Can you update constructor parameters here?

Reply:
thanks, updated